### PR TITLE
Updated librevenge recipe

### PR DESCRIPTION
**Pull request by miqlas**
On the road to LibreOffice

### CODE REVIEW

File: dev-libs/librevenge/librevenge-0.0.4.recipe [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+DESCRIPTION="librevenge is a base library for writing document import \
+filters. It has interfaces for text documents, vector graphics, spreadsheets \
+and presentations."
+HOMEPAGE="http://sourceforge.net/p/libwpd/wiki/librevenge/"
+COPYRIGHT="Fridrich Strba
+	Laurent Alonso
+	David Tardon"
+LICENSE="GNU LGPL v2.1
+	MPL v2.0"
+REVISION="2"
```
Comment (diversys) [inline dev-libs/librevenge/librevenge-0.0.4.recipe]: If it's a new recipe why revision is set to 2?

File: dev-libs/librevenge/librevenge-0.0.4.recipe [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libboost_filesystem$secondaryArchSuffix
+	devel:libboost_system$secondaryArchSuffix
+	devel:libcppunit$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:libtoolize$secondaryArchSuffix
```
Comment (diversys) [inline dev-libs/librevenge/librevenge-0.0.4.recipe]: Sort alphabetically

File: dev-libs/librevenge/librevenge-0.0.4.recipe [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	
+	# remove libtool library file
+	#rm $libDir/librevenge*.la
```
Comment (diversys) [inline dev-libs/librevenge/librevenge-0.0.4.recipe]: Why is it commented out?

**Comment by diversys** [comment]
I think you can `git mv librevenge-0.0.1.recipe librevenge-0.04.recipe`
Then your commit message will read:
librevenge: bump version to 0.0.4.

File: dev-libs/librevenge/librevenge-0.0.4.recipe [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+filters. It has interfaces for text documents, vector graphics, spreadsheets \
+and presentations."
+HOMEPAGE="http://sourceforge.net/p/libwpd/wiki/librevenge/"
+COPYRIGHT="Fridrich Strba
+	Laurent Alonso
+	David Tardon"
+LICENSE="GNU LGPL v2.1
+	MPL v2.0"
+REVISION="1"
+SOURCE_URI="http://prdownloads.sourceforge.net/libwpd/librevenge-$portVersion.tar.bz2"
```
Comment (fbrosson) [inline dev-libs/librevenge/librevenge-0.0.4.recipe]: You might wish to change this to:
```
SOURCE_URI="http://downloads.sourceforge.net/libwpd/librevenge-$portVersion.tar.bz2"
```
or
```
SOURCE_URI="http://downloads.sf.net/libwpd/librevenge-$portVersion.tar.bz2"
```

File: dev-libs/librevenge/librevenge-0.0.4.recipe [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+	"
+
+PROVIDES_devel="
+	librevenge${secondaryArchSuffix}_devel = $portVersion compat >= 0
+	devel:librevenge_0.0$secondaryArchSuffix = $portVersion compat >= 0
+	devel:librevenge_generators_0.0$secondaryArchSuffix = $portVersion compat >= 0
+	devel:librevenge_stream_0.0$secondaryArchSuffix = $portVersion compat >= 0
+	"
+REQUIRES_devel="
+	librevenge${secondaryArchSuffix} == $portVersion base
```
Comment (fbrosson) [inline dev-libs/librevenge/librevenge-0.0.4.recipe]: You can change this to:
```
	librevenge$secondaryArchSuffix == $portVersion base
```

File: dev-libs/librevenge/librevenge-0.0.4.recipe [hunk 6]
```diff
@@ -0,0 +1,83 @@
+SUMMARY="A helper library for REVerse ENGineered formats filters"
+DESCRIPTION="librevenge is a base library for writing document import \
+filters. It has interfaces for text documents, vector graphics, spreadsheets \
+and presentations."
+HOMEPAGE="http://sourceforge.net/p/libwpd/wiki/librevenge/"
+COPYRIGHT="Fridrich Strba
+	Laurent Alonso
+	David Tardon"
```
Comment (fbrosson) [inline dev-libs/librevenge/librevenge-0.0.4.recipe]: I've grabbed the source archive and analyzed its ChangeLog to find the copyright years.
I also noticed that 2 authors are missing in the COPYRIGHT variable. The following table gives the number of commits each major contributor has authored. Remark: I've discarded 12 authors which have contributed with only 1 to 7 commits each.

Commits | Authors
-------- | --------
842 | Fridrich Strba
191 | David Tardon
167 | William Lachance
150 | Marc Maurer
78 | Laurent Alonso & osnola (=Alonso)

Using grep and friends I find this:
```
COPYRIGHT="2004-2015 Fridrich Strba
	2012-2016 David Tardon
	2002-2006, 2010 William Lachance
	2002-2006 Marc Maurer
	2011-2015 Laurent Alonso"
```

**Pull request merged**